host.json

```diff
@@ -1,3 +1,11 @@
 {
-  "version": "2.0"
-}
+  "version": "2.0",
+  "extensions": {
+    "serviceBus": {
+      "messageHandlerOptions": {
+        "maxAutoRenewDuration": "00:01:00",
+        "maxConcurrentCalls": 5
```

Comment:
These settings apply to all functions, right? Could there by any impact on the other functions, or they all ran fine? Just wanted to double check.

Reply:
That file doesn't exists in that path anymore, but yes, it would apply to all functions I think.
For know, worker has this config, and the other functions use the default values.

When everything is in the same function, we will need to figure out if each function can have their own config, because for `sync` I think we can have more concurrent calls